assets/package.json

```diff
@@ -73,54 +67,32 @@
 		"use-memo-one": "^1.1.2"
 	},
 	"devDependencies": {
-		"@babel/core": "^7.14.8",
-		"@babel/plugin-transform-runtime": "^7.10.4",
-		"@babel/preset-react": "^7.10.4",
+		"@babel/preset-react": "^7.26.3",
+		"@storybook/addon-queryparams": "^6.2.9",
 		"@svgr/webpack": "^5.4.0",
+		"@testing-library/react": "^10.4.9",
+		"@testing-library/react-hooks": "^3.3.0",
```

Comment:
Should this not be in the `tests/js` package instead?

Reply:
Same as for the storybook dependency, we import these packages in our `.test.js` files, thus we need to have them here.